integration/main_test.go

```diff
@@ -13,6 +15,15 @@ import (
 	"time"
 )
 
+var atLeastGo15 bool = false
+
+func init() {
+	var major, minor int
+	var discard string
+	i, err := fmt.Sscanf(runtime.Version(), "go%d.%d%s", &major, &minor, &discard)
```

Comment:
go tip version does not follow this and it is greater than 1.5. but i am fine with not testing go tip.